sandy/decay.py

```diff
@@ -230,6 +237,78 @@ def get_br(self, with_uncertainty=True):
         else:
             return df.BR
 
+    def get_decay_energy(self, with_uncertainty=True):
+        """
+        Extract decay energy and its uncertainty into a dataframe.
+
+        Parameters
+        ----------
+        with_uncertainty : `bool`, optional, default is 'True'
+            makes the method return decay energies and uncertainties
+            if set equal True, or else return only the decay energies
+
+        Returns
+        -------
+        `pandas.DataFrame` or `pandas.Series`
+            dataframe with decay energy and associated uncertainty
+            series with decay energy if with_uncertainty = False
+
+        Examples
+        --------
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", [942400, 922350])
+        >>> rdd = sandy.DecayData.from_endf6(endf6)
+        >>> DE = rdd.get_decay_energy()
+        >>> print(DE)
+                              BE  Uncertainty
+        922350 alpha 4.46460e+06  1.63255e+05
+               beta  5.06717e+04  4.29163e+03
+               gamma 1.63616e+05  1.70801e+03
+               total 4.67889e+06  1.63320e+05
+        942400 alpha 5.24303e+06  3.63881e+04
+               beta  1.11164e+04  9.02572e+02
+               gamma 1.36292e+03  1.33403e+02
+               total 5.25551e+06  3.63995e+04
+
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", [942400, 922350])
+        >>> rdd = sandy.DecayData.from_endf6(endf6)
+        >>> DE = rdd.get_decay_energy(False)
+        >>> print(DE)
+        922350  alpha   4.46460e+06
+                beta    5.06717e+04
+                gamma   1.63616e+05
+                total   4.67889e+06
+        942400  alpha   5.24303e+06
+                beta    1.11164e+04
+                gamma   1.36292e+03
+                total   5.25551e+06
+        Name: BE, dtype: float64
+        """
+        DecayEnergy = []
+        DecayEnergyUncertainty = []
+        ZAM = []
+        for zam, dic in self.data.items():
+            DecayEnergy.extend([
+                dic['decay_energy']['alpha'],
+                dic['decay_energy']['beta'],
+                dic['decay_energy']['gamma'],
+                dic['decay_energy']['total'],
+                ])
+            DecayEnergyUncertainty.extend([
+                dic['decay_energy_uncertainties']['alpha'],
+                dic['decay_energy_uncertainties']['beta'],
+                dic['decay_energy_uncertainties']['gamma'],
+                dic['decay_energy_uncertainties']['total'],
+                ])
+            ZAM.append(zam)
+        name = ['alpha', 'beta', 'gamma', 'total']
+        df = pd.DataFrame(zip(DecayEnergy, DecayEnergyUncertainty),
+                          index=pd.MultiIndex.from_product([ZAM, name]),
+                          columns=['BE', 'Uncertainty'])
```

Comment:
Why is it called `BE`? Why here the uncertainty is called "uncertainty" and in `get_br` it is "DBR"? Be consistent! 
I suggest here using "E" and "DE".